pkgs/top-level/python-packages.nix

```diff
@@ -22212,6 +22212,25 @@ in modules // {
     };
   };
 
+  yolk = buildPythonPackage rec {
+    name = "yolk-${meta.version}";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/y/yolk/yolk-${meta.version}.tar.gz";
+      sha256 = "1f6xwx210jnl5nq0m3agh2p1cxmaizawaf3fwq43q4yw050fn1qw";
+    };
+
+    buildInputs = with self; [ nose ];
+
+    meta = {
+      description = "Command-line tool for querying PyPI and Python packages installed on your system.";
+      homepage = "https://github.com/cakebread/yolk";
+      version = "0.4.3";
```

Comment:
They removed it! https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

Good to know.